src/models/device.ts

```diff
@@ -2273,7 +2292,7 @@ const getDeviceModel = function (
 		startOsUpdate,
 
 		/**
-		 * @summary Get the OS update status of a device
+		 * @summary (Deprecated) Get the OS update status of a device. This will no longer return a useful status for runDetached=true updates.
```

Comment:
Use the @deprecated [tag](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#deprecated). You can add additional text to it.